### DISCUSSION TITLE
Change the name of the fork to something different

### DISCUSSION THREAD

**Discussion by asunekants** [anametologin/krohnkite#71] [Ideas]
If this fork will be maintained, IMO it would be good to change its name, to distinguish it from the original abandoned project, which is currently the only one that comes up in the Github search, and is the first result on Google.
What do you think?

**Comment by anametologin** [comment]
I'm not eager to rename the project. If I support this project, search engines will re-index it.

**Reply by asunekants** [reply]
Fair point. I just brought it up as a thought.
Thank you for your great work on it, by the way. FWIW - it makes my life slightly better :)
